### PR TITLE
Fiid 97/validacion inversion

### PR DESCRIPTION
**Pull request by francoFinniu**
- "Validacion de tu inversion modal" [FIID-97](https://finniu-tech.atlassian.net/browse/FIID-97)
-  add ValidationModal

Screenshot 2024-07-04 175932
Screenshot 2024-07-04 175904

### CODE REVIEW

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart [hunk 1]
```diff
@@ -50,19 +48,19 @@ class CarrouselSlide extends StatelessWidget {
...
-    required this.color,
   });
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
+    final isDarkMode = ref.watch(settingsNotifierProvider).isDarkMode;
     return SlideCarouselCard(
-      color: color,
+      color: isDarkMode ? 0xff292929 : 0xffE2F8FF,
```
Comment (harolCalzada) [inline lib/presentation/screens/catalog/widgets/carrousel_slide.dart:63]: not use magic numbers  , please define this numbers as a variable in colors file

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart [hunk 2]
```diff
@@ -79,15 +77,15 @@ class AnnualProfitability extends StatelessWidget {
             style: TextStyle(fontSize: 12),
             overflow: TextOverflow.ellipsis,
           ),
-          const Divider(
+          Divider(
             height: 2,
-            color: Color(primaryDark),
+            color: Color(isDarkMode ? 0xff828282 : 0xffA2E6FA),
```
Comment (harolCalzada) [inline lib/presentation/screens/catalog/widgets/carrousel_slide.dart:82]: not use magic numbers , please define this numbers as a variable in colors file

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart [hunk 3]
```diff
@@ -79,15 +77,15 @@ class AnnualProfitability extends StatelessWidget {
...
           ),
           Container(
             width: 245,
-            decoration: const BoxDecoration(
-              color: Color(annualCard),
-              borderRadius: BorderRadius.all(Radius.circular(20)),
+            decoration: BoxDecoration(
+              color: Color(isDarkMode ? 0xffA2E6FA : 0xffC3F1FF),
```
Comment (harolCalzada) [inline lib/presentation/screens/catalog/widgets/carrousel_slide.dart:87]: not use magic numbers , please define this numbers as a variable in colors file

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart [hunk 4]
```diff
@@ -128,19 +130,18 @@ class AnnualProfitability extends StatelessWidget {
...
-    required this.color,
   });
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
+    final isDarkMode = ref.watch(settingsNotifierProvider).isDarkMode;
     return SlideCarouselCard(
-      color: color,
+      color: isDarkMode ? 0xff174F79 : 0xffFFFFFF,
```
Comment (harolCalzada) [inline lib/presentation/screens/catalog/widgets/carrousel_slide.dart:144]: not use magic numbers , please define this numbers as a variable in colors file

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart [hunk 5]
```diff
@@ -157,7 +158,7 @@ class InvestedCapital extends StatelessWidget {
               borderColor: Colors.transparent,
               series: [
                 ColumnSeries(
-                  color: const Color(primaryLight),
+                  color: Color(isDarkMode ? 0xff0A2940 : 0xffA2E6FA),
```
Comment (harolCalzada) [inline lib/presentation/screens/catalog/widgets/carrousel_slide.dart:161]: not use magic numbers , please define this numbers as a variable in colors file

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart [hunk 6]
```diff
@@ -195,18 +196,18 @@ class InvestedCapital extends StatelessWidget {
...
-    required this.color,
   });
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final isSoles = ref.watch(isSolesStateProvider);
+    final isDarkMode = ref.watch(settingsNotifierProvider).isDarkMode;
     return SlideCarouselCard(
-      color: color,
+      color: isDarkMode ? 0xff0D3A5C : 0xffFFEEDD,
```
Comment (harolCalzada) [inline lib/presentation/screens/catalog/widgets/carrousel_slide.dart:210]: not use magic numbers , please define this numbers as a variable in colors file

**Review by harolCalzada — CHANGES_REQUESTED** [review]

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart (hunk 1 above)
Comment (francoFinniu) [reply lib/presentation/screens/catalog/widgets/carrousel_slide.dart]: colors are passed to variables, for light/dark theme

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart (hunk 2 above)
Comment (francoFinniu) [reply lib/presentation/screens/catalog/widgets/carrousel_slide.dart]: colors are passed to variables, for light/dark theme

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart (hunk 3 above)
Comment (francoFinniu) [reply lib/presentation/screens/catalog/widgets/carrousel_slide.dart]: colors are passed to variables, for light/dark theme

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart (hunk 4 above)
Comment (francoFinniu) [reply lib/presentation/screens/catalog/widgets/carrousel_slide.dart]: colors are passed to variables, for light/dark theme

File: lib/presentation/screens/catalog/widgets/carrousel_slide.dart (hunk 6 above)
Comment (francoFinniu) [reply lib/presentation/screens/catalog/widgets/carrousel_slide.dart]: colors are passed to variables, for light/dark theme

**Pull request merged**